boards/FSM/Src/App/states/App_FaultState.c

```diff
@@ -0,0 +1,34 @@
+#include "states/App_FaultState.h"
+#include "states/App_AllStates.h"
+#include "App_SharedMacros.h"
+
+void App_FaultStateRunOnEntry(struct StateMachine *const state_machine){
+    return;
+}
+
+void App_FaultStateRunOnTick1Hz(struct StateMachine *state_machine){
+    App_AllStatesRunOnTick1Hz(state_machine);
+}
+void App_FaultStateRunOnTick100Hz(struct StateMachine *state_machine){
+    App_AllStatesRunOnTick100Hz(state_machine);
+
+    struct FsmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+
+    //FORCE ACCELERATOR TO BE 0.
+    App_CanTx_SetPeriodicSignal_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
```

Comment:
There's a few other accelerator pedal signals that haven't been updated in this state (meaning they'll just hold their value from whatever they were last in drive state). 

Maybe this warrants rethinking setting the CAN signals in the individual modules? @joethrstn @Eddy13013

Reply:
Broadcasting should be done in all_states which should address this concern. Only difference in behavior is Mapped_pedal_position and coolant underflow fault which are state dependent. Those two pieces of functionality should utilize the specific states as req'd